IdentityServer/v6/docs/content/quickstarts/2_interactive.md

```diff
@@ -368,12 +368,12 @@ Next you will add sign-out to *WebClient*.
 
 To sign out, you need to 
 - Clear local application cookies
-- Make a roundtrip to *IdentityServer* using the OIDC protocol to clear its
+- Make a round-trip to *IdentityServer* using the OIDC protocol to clear its
```

Comment:
`round-trip` is debatable as acceptable forms are roundtrip, round trip, and round-trip.

Reply:
Would you suggest reverting those instances?